tests/functional/adapter/simple_seed/test_seed.py

```diff
@@ -0,0 +1,114 @@
+from pathlib import Path
+
+import pytest
+from dbt.tests.adapter.simple_seed.test_seed import (
+    TestBasicSeedTests as CoreTestBasicSeedTests,
+)
+from dbt.tests.adapter.simple_seed.test_seed import (
+    TestSeedConfigFullRefreshOff as CoreTestSeedConfigFullRefreshOff,
+)
+from dbt.tests.adapter.simple_seed.test_seed import (
+    TestSeedConfigFullRefreshOn as CoreTestSeedConfigFullRefreshOn,
+)
+from dbt.tests.adapter.simple_seed.test_seed import (
+    TestSeedCustomSchema as CoreTestSeedCustomSchema,
+)
+from dbt.tests.adapter.simple_seed.test_seed import (
+    TestSeedParsing as CoreTestSeedParsing,
+)
+from dbt.tests.adapter.simple_seed.test_seed import (
+    TestSeedSpecificFormats as CoreTestSeedSpecificFormats,
+)
+from dbt.tests.adapter.simple_seed.test_seed import (
+    TestSeedWithEmptyDelimiter as CoreTestSeedWithEmptyDelimiter,
+)
+from dbt.tests.adapter.simple_seed.test_seed import (
+    TestSeedWithUniqueDelimiter as CoreTestSeedWithUniqueDelimiter,
+)
+from dbt.tests.adapter.simple_seed.test_seed import (
+    TestSeedWithWrongDelimiter as CoreTestSeedWithWrongDelimiter,
+)
+from dbt.tests.adapter.simple_seed.test_seed import (
+    TestSimpleSeedEnabledViaConfig as CoreTestSimpleSeedEnabledViaConfig,
+)
+from dbt.tests.adapter.simple_seed.test_seed import (
+    TestSimpleSeedWithBOM as CoreTestSimpleSeedWithBOM,
+)
+from dbt.tests.util import copy_file, run_dbt
+
+from tests.functional.adapter.simple_seed.seeds import (
+    trino_seeds__expected_sql_create_table,
+    trino_seeds__expected_sql_insert_into,
+)
+
+
+class TrinoSetUpFixture:
+    @pytest.fixture(scope="class", autouse=True)
+    def setUp(self, project):
+        """Create table for ensuring seeds and models used in tests build correctly"""
+        project.run_sql(trino_seeds__expected_sql_create_table)
+        project.run_sql(trino_seeds__expected_sql_insert_into)
+
+
+class TestTrinoBasicSeedTests(TrinoSetUpFixture, CoreTestBasicSeedTests):
+    # TODO Trino currently does not support DROP TABLE CASCADE.
+    #  Dropping seed won't drop downstream models automatically.
```

Comment:
create an issue for that

Reply:
https://github.com/trinodb/trino/issues/19498

Reply:
in this repo as well

Reply:
https://github.com/starburstdata/dbt-trino/issues/369